api/handlers/datasources.py

```diff
@@ -39,3 +42,18 @@ def get_datasources(
     for datasource in datasources:
         result.append(datasource)
     return result
+
+
+@router.get(
+    "/api/datasources/{account}/{container}/image")
+def get_generated_image():
+    # Generate the image as place holder
+    image = Image.new("RGB", (200, 200), (255, 255, 255))
+    draw = ImageDraw.Draw(image)
+    draw.rectangle((50, 50, 150, 150), fill=(0, 0, 0))
+
+    image_data = io.BytesIO()
```

Comment:
I'm assuming this will be replaced with a subset of bytes from data file in the blob storage account in another user story?

Reply:
Yep has dependency on API sas token capability which is following